src/bmerge.c

```diff
@@ -298,7 +298,7 @@ void bmerge_r(int xlowIn, int xuppIn, int ilowIn, int iuppIn, int col, int thisg
     // ilow and iupp now surround the group in ic, too
   }
     break;
-  case STRSXP : {
```

Comment:
this `{` was here for consistency to other cases, and nice collapsing code

Reply:
by collapsing do mean in editor?  If we add `{` to all `case:` I don't think that's very standard.  I wrote my thinking here:  https://github.com/Rdatatable/data.table/pull/3610#issuecomment-497099447

Reply:
yes, collapsing code in IDE, makes it very convenient